js/controller.js

```diff
@@ -364,6 +366,16 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
       this.renderSkin();
     },
 
+    // NYM – enable closed captions when volume is off, or disable closed captions when volume is on
+    // for v4 players that default to displaying closed captions if available
+    onVolumeChangedToggleClosedCaptions: function (event, newVolue) {
+      this.state.persistentSettings.closedCaptionOptions['enabled']
+      if ((newVolue <= 0 && !this.state.closedCaptionOptions.enabled)
+        || (newVolue > 0 && this.state.closedCaptionOptions.enabled)) {
+        this.toggleClosedCaptionEnabled();
+      }
+    },
+
```

Comment:
RIGHT?!
was wondering about that too
volume/value hybrid?
Scrabble says `no`:
http://www.anagrammer.com/scrabble/volue